### PR TITLE
Add Sentieon possibilities to Sarek

### PR DESCRIPTION
**Pull request by MaxUlysse**
- Add sentieon possibilities to Sarek
  - preprocessing
  - variant calling with DNAseq, DNAscope and TNscope

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** [guidelines](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: conf/base.config [hunk 1]
```diff
@@ -75,4 +75,8 @@ process {
     container = {(params.annotation_cache && params.vep_cache) ? 'nfcore/sarek:dev' : "nfcore/sarekvep:dev.${params.genome}"}
     errorStrategy = {task.exitStatus == 143 ? 'retry' : 'ignore'}
   }
+  withLabel:sentieon {
+    beforeScript = {params.sentieon ? 'module load sentieon/201808.05' : ''}
```
Comment (MaxUlysse) [inline conf/base.config]: That should be moved to the `munin` config in nf-core/configs as it's highly specific.

**Comment by MaxUlysse** [comment]
Needs docs

**Comment by apeltzer** [comment]
Ping once you need a review!

**Comment by MaxUlysse** [comment]
Just one last commits with the docs and I'm ok

File: docs/output.md [hunk 2]
```diff
@@ -1,72 +1,97 @@
-# nf-core/sarek: Output
+# nf-core/sarek: Output <!-- omit in toc -->
```
Comment (apeltzer) [inline docs/output.md]: Uuh! Nice :-)

Comment (MaxUlysse) [reply docs/output.md]: I did just read the docs of the tool you advised to try ;-)

**Pull request merged**